packages/fxa-admin-panel/src/components/AppLayout/index.scss

```diff
@@ -30,3 +30,40 @@
 main {
   flex: 4;
 }
+
+// TODO: Setup Tailwind in admin-panel and remove these custom styles.
```

Comment:
I can't remember, is this in a new ticket?

Reply:
Yes, this is part of that ticket I'm making👍